src/main/java/com/epam/esm/dao/GiftCertificateTagDao.java

```diff
@@ -3,45 +3,57 @@
 import com.epam.esm.sql.SqlQueries;
 import com.epam.esm.entity.Tag;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.jdbc.core.JdbcTemplate;
-import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Repository;
+import javax.persistence.EntityManager;
 import java.util.List;
 import java.util.Set;
+import java.util.stream.Collectors;
 
-@Component
+@Repository
 public class GiftCertificateTagDao {
```

Comment:
I thought that we're going to get rid of this class. JPA's should do this logic dor us now